index.js

```diff
@@ -138,6 +139,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(markdown());
```

Comment:
Massive brain-fart. Somehow forgot I'd already merged this.

Don't worry, I'll do it.